applications/ExternalSolversApplication/custom_python/add_linear_solvers_to_python.cpp

```diff
@@ -52,6 +54,10 @@ void  AddLinearSolversToPython()
     typedef UblasSpace<double, CompressedMatrix, Vector> SpaceType;
     typedef UblasSpace<double, Matrix, Vector> LocalSpaceType;
     typedef LinearSolver<SpaceType,  LocalSpaceType> LinearSolverType;
+    typedef UblasSpace<std::complex<double>, boost::numeric::ublas::compressed_matrix<std::complex<double>>, boost::numeric::ublas::vector<std::complex<double>>> ComplexSpaceType;
```

Comment:
correct, just we might want to begin using the "using" keyword for this sort of things.

(i am NOT requiring to change though...just commenting)